pandas/core/window.py

```diff
@@ -780,8 +780,41 @@ def calc(x):
 
 class _Rolling_and_Expanding(_Rolling):
 
-    _shared_docs['count'] = """%(name)s count of number of non-NaN
-    observations inside provided window."""
+    _shared_docs['count'] = dedent(r"""
+    The %(name)s sum if it is a non-Nan value inside the window.
```

Comment:
Using the word `sum` is confusing here for a `count`. Can you reword to the effect of "count of non-`NaN` observations..."?

Reply:
"count of non-Nan observations inside the rolling window" would work?

Reply:
Yes

Reply:
@scriptomation be sure that you actually surround your reference to `NaN` in back ticks, so literally you would write non-\`NaN\` (capitalization matters too)